todo_app/tests/test_view_model.py

```diff
@@ -47,18 +51,27 @@ def test_items(view_model):
 
 
 def test_todo_items(view_model):
-    for item in view_model.todo_items:
-        assert item.status == "Not Started"
+    if len(view_model.todo_items) >= 1:
+        for item in view_model.todo_items:
+            assert item.status == "Not Started"
+    else:
+        assert view_model.todo_items == []
```

Comment:
I meant something like
```suggestion
    assert len(view_model.todo_items) == 1
    for item in view_model.todo_items:
        assert item.status == "To Do"
```
The reason why I bought this up was originally (before your recent changes) when your tests were run outside of VSCode the `in_progress_list` etc. environment variables were not set so would all be `None`. In todo_item.py `if card["idList"] == os.getenv("complete_list"):` would always be True because `None == None`, so all your cards would have the status `"Complete"`!
This test still passed if `view_model.todo_items` is an empty list, which is why `assert len(view_model.todo_items) == 1` would be an improvement, since it could catch similar bugs in the future.